src/main/java/duke/EventTask.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class EventTask extends Task {
+
+    protected LocalDateTime dateTime;
+    private String formattedDateTime;
+    private String oldDate;
+
+    /**
+     * Default constructor for Event.
+     * @param description Description of task.
+     * @param dateTime Datetime of task.
+     */
```

Comment:
Seems like there should be an empty line between description and parameter section. 🤔